Developers/Writing_Bears.rst

```diff
@@ -276,3 +276,148 @@ parameter in ``HiddenResult`` should be the instance of the Bear that yields
 this result (in this case ``self``), and second argument should be the content
 we want to transfer between the Bears. Here we use a list of strings as content
 but it can be any object.
+
+More Configuration Options
+--------------------------
+coala provides settings to further configure your bear according to your needs.
+Here is the list of all the settings you can use:
+
+- `LANGUAGES`_
+- `REQUIREMENTS`_
+- `INCLUDE_LOCAL_FILES`_
+- `CAN_DETECT and CAN_FIX`_
+- `BEAR_DEPS`_
+- `Other Settings`_
```

Comment:
@sils and metadata instead of settings here too? I dont know `Other Metadata` seems wrong to me.